lib/src/planner_layer/mod.rs

```diff
@@ -0,0 +1,243 @@
+pub mod error;
+pub mod logical_plan;
+pub mod query_planner;
+pub mod raw_expression;
+
+#[cfg(test)]
+mod tests {
+    use super::super::{
+        executor_layer::{expression::*, schema::*, table_manager::TableManager},
+        storage_layer::paged_storage::PagedStorage,
+    };
+    use super::{logical_plan::*, query_planner::QueryPlanner, raw_expression::*};
+
+    pub mod utility {
+        use super::*;
+        use std::process::Command;
+
+        #[cfg(test)]
```

Comment:
You don't need this pragma since the toplevel `#[cfg(test)]` already does the work